### ISSUE TITLE
Get picture data in pdf?

### ISSUE THREAD

**Issue by bigbird231** [modesty/pdf2json#256]
I want to get picture data in pdf.
now I can only get text data from pdf by pdf2json,is it possible to get picture data?

**Comment by reregaga** [comment]
No, still under development: https://github.com/modesty/pdf2json/pull/91#issuecomment-979430735
I vote for your proposal.

You can use other libs for getting images, for example [mupdf-js](https://github.com/andytango/mupdf-js) can convert pdf to svg and then search in svg `image` tags.
